README.md

```diff
@@ -169,7 +169,7 @@ Note: Upstart/SysV init based OS types are not supported.
 - Network Plugin
   - [cni-plugins](https://github.com/containernetworking/plugins) v1.2.0
   - [calico](https://github.com/projectcalico/calico) v3.25.1
-  - [cilium](https://github.com/cilium/cilium) v1.13.3
+  - [cilium](https://github.com/cilium/cilium) v1.14.7
```

Comment:
Wait this is wrong 1.14.7 doesn't exist 🤔, is it the cilium cli or something?

Reply:
You're right, the latest version is [1.13.4](https://github.com/cilium/cilium/releases/tag/v1.13.4)